lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
Relating to this -- how would this feature deal with the question of certificate generation (given the policy differences)?